system/zfs.py

```diff
@@ -237,6 +237,36 @@
 
 import os
 
+def to_bytes(value):
```

Comment:
these functions would belong in a lib/ansible/module_utils/basic.py (see existing pretty_bytes)